src/request.js

```diff
@@ -30,6 +30,51 @@ export function hashPayload(payload) {
         .substr(0, 8);
 }
 
+/**
+ * Specifies required and optional fields for constructing a {@link Request}.
+ *
+ * @typedef {Object} RequestOptions
```

Comment:
`RequestOptions` is already defined at the end on line 273. If you want, move to the top

Reply:
Weird. I definitely did not write all that. Probably a bad merge of your rebased branch.